source/mainnet/smart-contracts/tutorials/voting/voting-dapp.rst

```diff
@@ -37,116 +39,109 @@ need for the user to host their own Concordium node.
 
-    .. image:: ../wCCD/images/wCCD_tutorial_18.png
-        :width: 100 %
+.. image:: ../wCCD/images/wCCD_tutorial_18.png
+    :width: 100 %
 
-    .. note::
+.. note::
 
-        The puzzle icon at the top right of the ``Chrome`` browser allows you to manage your browser extensions.
-        You can enable pinning of the |bw|.
+    The puzzle icon at the top right of Google Chrome allows you to manage your browser extensions.
+    Here, you can enable pinning of the wallet extension to always have it handy.
 
-        .. image:: ../wCCD/images/wCCD_tutorial_13.png
-            :width: 30 %
+    .. image:: ../wCCD/images/wCCD_tutorial_13.png
+        :width: 30 %
 
-    Create a new account on testnet by going through the setup steps of the |bw|.
-    You have to choose a password to secure your |bw|.
-    This password is needed to log in to your |bw|.
+Create a new account on testnet by going through the setup steps.
+You have to choose a password to secure the wallet.
+This password is needed to log in to the wallet.
 
-    .. image:: ../wCCD/images/wCCD_tutorial_15.png
-        :width: 30 %
+.. image:: ../wCCD/images/wCCD_tutorial_15.png
+    :width: 30 %
 
-    The |bw| creates a unique secret recovery phrase. Write down the secret recovery phrase
-    and keep it in a safe place to be able to recover your accounts in case
-    you lose access to your device.
+The wallet creates a unique secret recovery phrase. Write down the secret recovery phrase
+and keep it in a safe place to be able to recover your accounts in case
+you lose access to your device.
 
-    You have completed the setup. Check that your |bw| is connected to the testnet.
+You have now completed the setup. Please verify that your wallet is connected to the Concordium Testnet.
 
-    .. image:: ../wCCD/images/wCCD_tutorial_16.png
-        :width: 30 %
+.. image:: ../wCCD/images/wCCD_tutorial_16.png
+    :width: 30 %
 
-    .. image:: ../wCCD/images/wCCD_tutorial_17.png
-        :width: 30 %
+.. image:: ../wCCD/images/wCCD_tutorial_17.png
+    :width: 30 %
 
-    Before you can create a new account. You need to create an identity.
+Before you can create a new account, you need to create an identity. You can do this easily on testnet via the Concordium testnet ID provider.
 
-    .. image:: ../wCCD/images/wCCD_tutorial_19.png
-        :width: 30 %
+.. image:: ../wCCD/images/wCCD_tutorial_19.png
+    :width: 30 %
 
-    .. image:: ../wCCD/images/wCCD_tutorial_20.png
-        :width: 30 %
+.. image:: ../wCCD/images/wCCD_tutorial_20.png
+    :width: 30 %
 
-    .. image:: ../wCCD/images/wCCD_tutorial_21.png
-        :width: 30 %
+.. image:: ../wCCD/images/wCCD_tutorial_21.png
+    :width: 30 %
 
-    .. image:: ../wCCD/images/wCCD_tutorial_22.png
-        :width: 30 %
+.. image:: ../wCCD/images/wCCD_tutorial_22.png
+    :width: 30 %
 
-    You are ready now to create a new account on testnet.
+Finally, make a new account on testnet.
```

Comment:
```suggestion
Finally, create a new account on testnet.
```

Reply:
Quite the nitpick but fine :sweat_smile: